src/Main.java

```diff
@@ -0,0 +1,48 @@
+
+import java.util.*;
+
+public class Main {
```

Comment:
I will suggest have only  2 classes Member and Chat(and not Chatting) and Chat class will have main method, it is the way I will do it, but this is also acceptable :)